PoiPoiWeatherForeCastGoogleColab.ipynb

```diff
@@ -40,7 +40,7 @@
         "\n",
         "def readHistoryDataExtra():\n",
         "    DataFrame_columns=['Name','Date time','Maximum Temperature','Minimum Temperature','Temperature','Wind Chill','Heat Index','Precipitation','Snow','Snow Depth','Wind Speed','Wind Direction','Wind Gust','Visibility','Cloud Cover','Relative Humidity','Conditions']\n",
-        "    DataFrame = pd.read_csv('Adatforrás/Budapest/history_data_extra.csv', sep=',', skiprows=1, names=DataFrame_columns)\n",
+        "    DataFrame = pd.read_csv('AFP-PoiPoi-Weather-uni/Adatforrás/Budapest/history_data_extra.csv', sep=',', skiprows=1, names=DataFrame_columns)\n",
```

Comment:
ez lehet működik ./Adatforrás (notice the dot) de colab/windows/linux 3way intersection lehet összeverekedik az elérési út formátumon 😟